### DISCUSSION TITLE
Notification & Unread Badge

### DISCUSSION THREAD

**Discussion by madeindra** [community/community#3220] [Discussions] [answered]
Hi!

I've noticed there was some discussion that happened in my repo, but I did not get any notification and I did not know it until I open the menu.

I think it will be nice to implement notification for new discussion or simply a badge to show number of unread/ongoing discussion.

Thanks!

**Comment by evi-liu** [comment, marked as answer]
If you're subscribed to the repo, then you should get notifications for all Discussions opened. Additionally, have you tried enabling Discussions in `Custom notifications`?
![image](https://user-images.githubusercontent.com/8809849/106335750-45012a80-6242-11eb-9b66-ba2529174b41.png)

**Reply by madeindra** [reply]
@evi-liu Thanks for this! I will try your suggestion

**Reply by esatterwhite** [reply]
I think the notion of unread discussions - to be able to see how many discussions have unread / non-viewed replies since your last visit is still the thing I would want.

especially for private repo and organization with many dozens to hundres of repositories. The notifications section can really become a bit of noise

![image](https://user-images.githubusercontent.com/148561/108106067-16ad8880-7053-11eb-9e54-f01db3342320.png)

having the number inline on the tab with issues and prs would be fantastic.
